JustSaying.AwsTools.IntegrationTests/JustSaying.AwsTools.IntegrationTests.csproj

```diff
@@ -0,0 +1,33 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\build\common.props" />
+  <PropertyGroup>
+    <TargetFramework>net46</TargetFramework>
+  </PropertyGroup>
+  <ItemGroup>
+    <None Include="App.config" />
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\JustSaying\JustSaying.csproj" />
+    <ProjectReference Include="..\JustSaying.AwsTools\JustSaying.AwsTools.csproj" />
+    <ProjectReference Include="..\JustSaying.Messaging\JustSaying.Messaging.csproj" />
+    <ProjectReference Include="..\JustSaying.Models\JustSaying.Models.csproj" />
+    <ProjectReference Include="..\JustSaying.TestingFramework\JustSaying.TestingFramework.csproj" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.0.0" />
+    <PackageReference Include="JustBehave" Version="1.0.0.12" />
+    <PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
+    <PackageReference Include="NSubstitute" Version="2.0.2" />
+    <PackageReference Include="NUnit" Version="3.6.1" />
+    <PackageReference Include="AWSSDK.Core" Version="3.3.8.2" />
+    <PackageReference Include="AWSSDK.SQS" Version="3.3.1.7" />
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="1.1.1" />
+    <PackageReference Include="AWSSDK.SimpleNotificationService" Version="3.3.0.7" />
+    <PackageReference Include="dotnet-test-nunit" Version="3.4.0-beta-3" />
```

Comment:
No, csproj doesn't support tests, so this project (dotnet-test-nunit) was a complete waste of time. We have to go back to vs test adapters, which I'm investigating, the nunit one isn't compatible with .NET Core, I have mixed feelings about that because we are targeting full framework any way for tests (but I don't want to be).

Locally I have ported to xunit (adding shouldly too, as the xunit assertions are very lacking, controversial!), I'll do some more tests to see if we can defer this migration.